th2-service/templates/ingress-rules.yaml

```diff
@@ -17,11 +18,11 @@ spec:
   - http:
   {{- end }}
       paths:
-      - path: /${SCHEMA_NAMESPACE}/backend/(.*)
+      - path: /${SCHEMA_NAMESPACE}/backend($|/.*)
```

Comment:
won't work for the second level endpoints, but it is not required for non-human urls, please remove